cli/internal/packager/create.go

```diff
@@ -119,10 +119,16 @@ func addLocalAssets(tempPath componentPaths, assets config.ZarfComponent) {
 		// Load all specified git repos
 		for _, url := range assets.Repos {
 			matches := strings.Split(url, "@")
-			if len(matches) < 2 {
-				logrus.WithField("remote", url).Fatal("Unable to parse git url. Ensure you use the format url.git@tag")
+			// Pull all of the references if there is no `@` in the string
+			pullAll := len(matches) == 1
+			cloneDir := git.Pull(url, tempPath.repos, pullAll)
+			if !pullAll {
```

Comment:
I'm fine with it either way but I think this might make a lot of sense to put at the end of the `git.Pull()` call.